webextensions/javascript-apis.json

```diff
@@ -4342,6 +4342,9 @@
                   "version_added": false
                 },
                 "firefox": {
+                  "notes": [
+                    "Only available at 'menus.ContextType', not at 'contextMenus.ContextType'."
```

Comment:
How about `"Only available at <code>menus.ContextType</code>, not at <code>contextMenus.ContextType</code>."` ?

Reply:
The WE docs don't include markup in notes, because they were mostly written before it became possible (also because I'm still not very reconciled to it). But OK :).